taipy/core/_orchestrator/_dispatcher/_job_dispatcher.py

```diff
@@ -96,11 +94,10 @@ def _execute_job(self, job: Job):
 
     def _execute_jobs_synchronously(self):
         while not self.orchestrator.jobs_to_run.empty():
-            with self.lock:
-                try:
-                    job = self.orchestrator.jobs_to_run.get()
-                except Exception:  # In case the last job of the queue has been removed.
-                    self._logger.warning(f"{job.id} is no longer in the list of jobs to run.")
+            try:
+                job = self.orchestrator.jobs_to_run.get()
+            except Empty:
+                pass
```

Comment:
Well spotted. We cannot log the job id!

However, I would still except all Exceptions, just to be sure that we keep running if any other exception is raised.

```suggestion
            except Exception:
                pass
```